src/main/java/core/basesyntax/db/StorageImpl.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
Consider making the 'fruits' map private and non-static, and provide methods to interact with the data in the map.